.github/workflows/ci.yml

```diff
@@ -171,21 +161,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           cache: 'pip'
-          cache-dependency-path: setup.cfg
+          cache-dependency-path: pyproject.toml
 
       - name: Install Python dependencies
-        if: runner.os != 'Windows'
         run: |
           pip install --upgrade pip
-          pip install .
-          pip install ".[test]"
-
-      - name: Install Python dependencies Windows
-        if: runner.os == 'Windows'
-        run: |
-          python.exe -m pip install --upgrade pip
-          python.exe -m pip install .
-          python.exe -m pip install ".[test]"
+          pip install git+https://git@github.com/Deltares/xmipy@develop
```

Comment:
is this still necessary? []v1.3.0](https://github.com/Deltares/xmipy/releases/tag/v1.3.0) was released last week.

Reply:
I could see that it is a good idea for modflowapi CI to use `xmipy@develop`. @mjr-deltares and @Hofer-Julian what do you think?

Reply:
I agree that it makes sense to use develop in the CI